Sources/PackageGraph/PackageGraphLoader.swift

```diff
@@ -150,11 +148,11 @@ private func createResolvedPackages(
         // Make sure these module names are unique in the graph.
         let dependencyModuleNames = dependencies.lazy.flatMap{ $0.modules }.map{ $0.name }
         if let duplicateModules = dependencyModuleNames.duplicates(modules.lazy.map{$0.name}) {
-            errors.append(ModuleError.duplicateModule(duplicateModules.first!))
+            engine.emit(ModuleError.duplicateModule(duplicateModules.first!))
```

Comment:
FWIW, Swift uses `diagnose` for this:
https://github.com/apple/swift/blob/master/include/swift/AST/DiagnosticEngine.h#L587

There may be some value in using consistent terminology.